app/src/androidTest/java/org/mozilla/focus/activity/screenshots/SettingsScreenshots.java

```diff
@@ -133,8 +134,12 @@ public void takeScreenShotsOfSettings() throws Exception {
         onView(withText(getString(R.string.preference_autocomplete_custom_summary)))
                 .perform(click());
         /* Add custom URL */
-        onView(withText(getString(R.string.preference_autocomplete_subitem_customlist)))
+        final String key = InstrumentationRegistry
+                .getTargetContext()
+                .getString(R.string.pref_key_screen_custom_domains);
+        onData(withKey(key))
```

Comment:
Why do we need to change this away from `onView(withText(`? The PR doesn't provide any context.

`onData` says it's useful for AdapterViews but we're in a preference (which may actually use an AdapterView under the hood though).

Reply:
@mcomella Oh, sorry about missing the background info on this. so for some locales, 'Custom URL' and 'Custom URLs' have identical text. so for those locales, the screenshot test fails if we only use the text to locate the identifier. #1888 is the bug that I raised for this.